_templates/development.html

```diff
@@ -39,23 +39,23 @@ <h2 id="more">{% translate more %}</h2>
 
 <ul class="devprojectlist">
   <li><a href="https://github.com/etotheipi/BitcoinArmory">Armory</a> - A Bitcoin client with enhanced security features.</li>
-  <li><a href="http://bfgminer.com">BFGMiner</a> - Modular Bitcoin mining software.</li>
-  <li><a href="https://code.google.com/p/bitcoinj/">bitcoinj</a> - A Java implementation of a Bitcoin client-only node used in thin SPV Bitcoin clients.</li>
-  <li><a href="https://github.com/schildbach/bitcoin-wallet/">Bitcoin Wallet for Android</a> - A thin SPV Bitcoin client for mobiles.</li>
-  <li><a href="https://github.com/bitsofproof/supernode">Bits of Proof Enterprise Bitcoin Server</a> - A modular implementation of the Bitcoin protocol in Java.</li>
+  <li><a href="http://bfgminer.com">BFGMiner</a> - A modular ASIC/FPGA Bitcoin miner.</li>
```

Comment:
Not sure what the purpose of this change is. BFGMiner supports CPU/GPU/FPGA/ASIC, but only ASIC makes sense for Bitcoin usually.